README.md

```diff
@@ -12,15 +12,20 @@ A Quick-start Guide for people that want to **D**o **W**hat **Y**ou **L**ove!
 The world is not perfect.  
 We have all worked somewhere we _didn't **love**_. <br />
 We have _all_ felt the _frustration_ of using products that have flaws
-(and been unable to fix them).   
-**The aim of _dwyl_ is to address *both* of these issues _simultaneously_.**
-
-If you'd like to know more, _please **read our [manifesto](/manifesto.md)**_.  
+that _we_ were unable to fix ourselves.   
+We have all experiened being in an environment that
```

Comment:
I felt that adding this sentence brought this a bit more 'together' and up to date. But it can easily be removed